src/wwwroot/app/directives/thesaurus/thesaurus.directive.html

```diff
@@ -0,0 +1,46 @@
+<div class="errorMessage" ng-bind="message" ng-disabled="!message"></div>
+<table>
+   <caption ng-bind="::structure.thesaurusName | translate"></caption>
+   <tr><th ng-repeat="field in ::structure.fields | filter : isShowField" ng-bind="::field.label"></th></tr>
```

Comment:
so, you call `field in ::structure.fields | filter : isShowField` three times
it will be good for performance, and for codestyle, if you do this once in your controller